constellations/gemini-admin/src/classes/classRow.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+require("./classRow.styl");
+import React from "react";
+import moment from "moment";
+import { Link } from "react-router-dom";
+
+export class ClassRow extends React.Component {
+    render() {
+        const row = this.props.row;
+        const unpublished = this.props.unpublished;
+
+        return (
+            <div id={unpublished ? "unpublished" : ""}>
+                {renderCheckbox(
+                    row.classId,
+                    this.props.collapsed,
+                    unpublished,
+                    this.props.selected,
+                    this.props.onSelectRow
+                )}
+                <span className="small">
+                    {unpublished ? "Unpublished" : "Published"}
+                </span>
+                <span className="large">{row.classId}</span>
+                <span className="small">{row.locationId}</span>
+                <span className="medium">
+                    {moment(row.startDate).format("M/D/YYYY")}
+                    {" - "}
+                    {moment(row.endDate).format("M/D/YYYY")}
+                </span>
+                <span className="large">{row.times}</span>
+                <Link to={"/class/" + row.classId + "/edit"}>Edit</Link>
+            </div>
+        );
+    }
+}
+
+function renderCheckbox(
+    classId,
+    collapsed,
+    unpublished,
+    selected,
+    onSelectRow
+) {
```

Comment:
did npm prettify do this?
I would prefer if all the variables were on the same line provided they don't go over 80 characters total.

Reply:
Yeah `npm run prettify` automatically did this

Reply:
ok it's fine then